heft-plugins/heft-jest-plugin/src/JestPlugin.ts

```diff
@@ -591,6 +591,7 @@ export default class JestPlugin implements IHeftTaskPlugin<IJestPluginOptions> {
       rootDir: buildFolderPath,
 
       silent: options.silent || false,
```

Comment:
This should be set to false if one of the CLI parameters is provided

Reply:
Compare with this discussion: https://github.com/jestjs/jest/issues/4458#issuecomment-328402758

Reply:
🤔  But that discussion is about the `--silent` CLI parameter.

Heft doesn't expose "silent" via the CLI, and I think this is a better design.  As stated above, it seems like a poor practice for unit tests to print to `console.log()` during successful operation, because it just pollutes the CI logs with useless noise.  Although it's probably often difficult for engineers to prevent product code from calling `console.log()` when invoked from a test.

Our recommended use cases might be like:

- Specify `silent=true` in **jest.config.json** to discard all `console.log()` output.  If something is failing, use `heft test --verbose` to print the output so we can inspect it.  OR
- Specify `silent=false` in **jest.config.json** so that `console.log()` is always shown for that project.  Then enable an ESLint rule that forbids `console.log()` from ever getting committed to `main`.  In this case, it is only ever used for investigative purposes.

Based on this, I think I agree that `--verbose` should override `silent=true`

Reply:
Hmmm... in our current implementation, we actually hard-wire `silent=false` ignoring **jest.config.json**

Reply:
Fixed in 96db6e6ac6e2f4fec7575f3c80efb107f8b44b67